packages/app/src/cli/services/dev/processes/dev-session-status-manager.ts

```diff
@@ -0,0 +1,36 @@
+import {deepCompare} from '@shopify/cli-kit/common/object'
```

Comment:
Coverage looks to have dropped, are the tests properly covering this?

Reply:
Totally forgot to add tests for the manager 🤦